contrib/aws/aws-sdk-go/aws/option.go

```diff
@@ -0,0 +1,26 @@
+package aws // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/aws/aws-sdk-go/aws"
+
+const (
+	awsAgentTag     = "aws.agent"
+	awsOperationTag = "aws.operation"
+	awsRegionTag    = "aws.region"
+)
+
+type config struct {
+	// when set to the empty string, the service name will be inferred based on
+	// the request to AWS
+	serviceName string
+}
+
+// Option represents an option that can be passed to Dial.
+type Option func(*config)
+
+func defaults(cfg *config) {
```

Comment:
Let's remove this function.

![asd](https://media.giphy.com/media/5x89XRx3sBZFC/giphy.gif)